benches/ldbc-graphalytics/main.rs

```diff
@@ -238,6 +243,74 @@ fn main() {
             let kevps = evps / 1000.0;
             println!("achieved {kevps:.02} kEVPS ({evps:.02} EVPS)");
 
+            if config.output_csv {
+                let results_file_already_exists = Path::new(LDBC_RESULT_CSV).is_file();
+                // If the file exists, we append another row, this is different
+                // from other benchmarks that can run everything with a single
+                // invocation:
+                let file = OpenOptions::new()
+                    .write(true)
+                    .append(results_file_already_exists)
+                    .create(!results_file_already_exists)
+                    .open(LDBC_RESULT_CSV)
+                    .expect("failed to open results csv file for writing");
+                    let mut csv_writer = csv::Writer::from_writer(file);
```

Comment:
```suggestion
                let mut csv_writer = csv::Writer::from_writer(file);
```

Reply:
It's weird that cargo fmt didn't format this (or complain about it) -- does it not check things in benches?

Reply:
(I did run `cargo fmt` and noticed it didn't touch/fix formatting issue in this file)

Reply:
(Updating cargo fmt seems to have helped its formatting again)